web/app/help/operations/find-optimal-spanning-tree.asciidoc

```diff
@@ -0,0 +1,28 @@
+### Find optimal spanning tree
```

Comment:
But please take a look at this. I've added this whole box after your approval. Thanks!